torchaudio/models/wavernn.py

```diff
@@ -324,3 +346,24 @@ def forward(self, waveform: Tensor, specgram: Tensor) -> Tensor:
 
         # bring back channel dimension
         return x.unsqueeze(1)
+
+
+def wavernn(checkpoint_name: str) -> WaveRNN:
+    r"""Get pretrained WaveRNN model.
+
+    Args:
+        checkpoint_name (str): The name of the checkpoint to load. Available checkpoints:
+
+            - wavernn_10k_epochs_8bits_ljspeech:
```

Comment:
```suggestion
            - ``"wavernn_10k_epochs_8bits_ljspeech"``
```

Reply:
Thanks for point it out. I've fixed them [here](https://github.com/pytorch/audio/pull/1612/files#diff-7f4ad259da182d3840fe4c75dd0c0b7b9fe30f31503df0eff87b1e5c9378cac4R357).